python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -39,7 +39,7 @@ class LocalFile(IGangaFile):
                                      })
     _category = 'gangafiles'
     _name = "LocalFile"
-    _exportmethods = ["location", "remove", "accessURL", "processOutputWildcardMatches", "hasMatchedFiles"]
+    _exportmethods = ["location", "remove", "accessURL"]
```

Comment:
No, I think it's ok overall. They *should* be private implementation details but for some reason (the commit message for 61666d2dd6acd4edf5374bb9985cc161f5c971d3 doesn't give any justification) they were exposed as public API. The problem with removing functions from the public API is anyone's scripts using them will now break. In reality I doubt that anyone will be using them so they should be fine to remove.